FOR_ANALYSTS.md

```diff
@@ -48,6 +48,31 @@ The following additional parameters can be specified for a dataset call:
 
 For more information on how to use and manipulate the resulting data see the [pandas documentation](http://pandas.pydata.org/).
 
+#### Download Entire Database (Bulk Download)
+
+You can download all the data of a database in a single call. The following will download the entire EOD database as a zip file to your current working directory:
```

Comment:
Im not sure what the best wording may be, but i copied "all the data" from here: https://www.quandl.com/docs/api#entire-database maybe it should be "all the data in a database..."